sensing/preprocessor/pointcloud/pointcloud_preprocessor/package.xml

```diff
@@ -30,6 +30,9 @@
   <depend>libopencv-dev</depend>
   <depend>libpcl-all-dev</depend>
 
+  <test_depend>ament_lint_auto</test_depend>
+  <test_depend>ament_cmake_cppcheck</test_depend>
```

Comment:
Esteve let me know that this should be`ament_cppcheck`, without the `cmake`.

Reply:
Just saw your comment on the other PR – in that case nevermind, `ament_cmake_cppcheck` seemed the right option to me anyway, since that is the one included by `ament_lint_common`.